.github/workflows/lintprettier.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
wydaje mi się, że lint/prettier to powinien być jeden z kroków builda, nie widzę celu w odpalaniu tego jako oddzielna akcja.
`npm run lint` i `npm run prettier` powinny się odpalać jako kroki przed `npm run build` i w razie czego jeśli któraś z nich nie przejdzie to wywalać dalsze budowanie projektu.